src/SourceBuild/content/eng/pipelines/ci.yml

```diff
@@ -14,7 +14,6 @@
 # - dotnet-source-build (internal)
 #   - PR: ultralite build
 #   - CI: release/* and internal/release/* only, every batched commit, full build
-#   - CI: main only, every batched commit, lite build

```

Comment:
I'm confused by this. We still want the lite build to run on every commit. That's what I interpret this comment as.

Reply:
I was expecting this line was just moving to:

#  - dotnet-source-build-lite (internal)
#   - CI: main only, every batched commit, lite build

Reply:
Thanks, we discussed this offline, pushed the new changes.